release/upload.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
```

Comment:
Note that `/usr/bin/env` is not guaranteed by posix either, contrary to popular belief of nix users.